### ISSUE TITLE
[BUG] Unexpected HTML in wysiwyg cke text editor widget

### ISSUE THREAD

**Issue by TarImre** [darkreader/darkreader#4978]
# Bug Report

## Describe the bug
Unexpected additional HTML is inserted into WYSIWYG CKE text editor widget when I formatted a selected text to bold.

## Steps to reproduce
  - Type a "hello" text into a CKE text editor widget
  - Select the "hello" word and make it bold
  - Click on the `Source` button to see the generated text source
  - You can see a lot of unexpected HTML tags. 

## Expected behavior
Do not touch the HTML in the text editor.  
```
<p><strong>hello</strong></p>
```

## Actual behavior
```
<p><strong>hello</strong></p>

<div id="simple-translate">
<div>
<div class="isShow simple-translate-button" style="background-image:url(&quot;moz-extension://5d58f3c2-2992-459c-9cd0-dcab506eaf5e/icons/512.png&quot;); height:22px; left:-17px; top:44px; width:22px">&nbsp;</div>

<div class=" simple-translate-panel" data-darkreader-inline-bgcolor="" style="--darkreader-inline-bgcolor:#181a1b; background-color:#ffffff; font-size:13px; height:200px; left:0px; top:0px; width:300px">
<div class="simple-translate-result-wrapper" style="overflow:hidden">
<div class="simple-translate-move" draggable="true">&nbsp;</div>

<div class="simple-translate-result-contents">
<p class="simple-translate-result" data-darkreader-inline-color="" style="--darkreader-inline-color:#e8e6e3; color:#000000">&nbsp;</p>

<p class="simple-translate-candidate" data-darkreader-inline-color="" style="--darkreader-inline-color:#9f978b; color:#737373">&nbsp;</p>
</div>
</div>
</div>
</div>
</div>

```

## Screenshots
![image](https://user-images.githubusercontent.com/10563321/107011545-116d5580-6798-11eb-8b21-9e50c63ef722.png)

## System Information
- OS: Windows 10
- Browser: Firefox 84.0.2 (64-bit)
- Dark Reader Version: 4.9.16
- CKE: 4.16.0

**Comment by Gusted** [comment]
Hi @TarImre!

Do you have an website link?

**Comment by TarImre** [comment]
Unfortunately, it occurs on a site that is part of a corporate network. Not accessible from the outside.

**Comment by TarImre** [comment]
After I investigated the unexpected HTML content I saw that there is another extension which can be blamed for this situation. The "Simple Translate" extension. So I checked it and I found that the problem only appears if the button of the "Simple Translate" is shown in the text area when I click the bold format button.

![kép](https://user-images.githubusercontent.com/10563321/107088480-4b247780-67fd-11eb-8ab3-92076e21878c.png)


(The "Simple Translate" extension provides an easy way to translate a selected text by a popup button. Quoted from the extension description: "When you select text, the translation button pops up and the translation is displayed instantly.")

Eventually, I found a workaround, when an I select a text and the "Simple Translate" button pops up then I press the ESC and the button disappear so after this I can bold the text without any unexpected additional HTML insertion.
